ungrib/src/rrpr.F

```diff
@@ -1399,4 +1412,43 @@ subroutine fix_ruc_soilm (ix, jx)
 
 end subroutine fix_ruc_soilm
  
-
+subroutine fix_gefs_landmask (ix, jx, plvl)
+! Set LANDSEA to values based on soil temp. Assumes fix_gfs_miss has already been called.
+! Needed for September 23, 2020 changes to GEFS. 
+! Earlier GEFS files are unmodified. Must be called after fix_gfs_miss
+! plvl must always be 200100.
+!
+  use storage_module
+  use module_debug
+  implicit none
+  integer :: ix, jx, i, j, k
+  real :: plvl
+  real, allocatable, dimension(:,:) :: f, sea
+!
+! If LANDN is present (not currently the case) then exit
+  if ( is_there(200100, 'LANDN') ) then
+    return
```

Comment:
Before returning, would it be worth writing out a log message to indicate that nothing is being done in this routine because the `LANDN` exists? I admit that I don't know what `LANDN` is or how its existence would obviate the need to fix the `LANDSEA` field.

Reply:
I don't know enough to add a reasonable print. Let's leave it out.